tests-functional/clients/status_backend.py

```diff
@@ -119,6 +127,45 @@ def create_account_and_login(self, display_name="Mr_Meeseeks", password=user_1.p
         }
         return self.api_valid_request(method, data)
 
+    def restore_account_and_login(self, display_name="Mr_Meeseeks", user=user_1):
+        method = "RestoreAccountAndLogin"
+        data = {
+            "rootDataDir": "/..",
```

Comment:
let's use `/`, without `..`, 
we can also create a new folder using timestamp but it's not that important since container is always recreated for every new run

Reply:
In fact, it would be great to use some temp directory for this